daal4py/sklearn/neighbors/knn.py

```diff
@@ -209,3 +213,60 @@ def predict(self, X):
         else:
             logging.info("sklearn.neighbors.KNeighborsClassifier.predict: " + method_uses_sklearn)
             return super(KNeighborsClassifier, self).predict(X)
+
+    def predict_proba(self, X):
```

Comment:
Why this method is needed? It's optimized through self.kneighbors

Reply:
Removed